packages/fxa-auth-server/lib/senders/emails/templates/lowRecoveryCodes/index.mjml

```diff
@@ -5,21 +5,39 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-header">
-      <span data-l10n-id="codes-reminder-title">Low recovery codes remaining</span>
+      <% if (numberRemaining < 1 ) { %>
```

Comment:
Nit, do you mind if we explicitly check for `=== 0` and `=== 1` instead of `< 1` and `else if < 2`? It reads as more clear to me, though the check on L21 seems fine since it accounts for 0 and 1.

Reply:
Of course! Great idea